data/tlg0085/tlg002/tlg0085.tlg002.1st1K-grc1.xml

```diff
@@ -28,16 +28,15 @@
         <biblStruct>
           <monogr>
             <author>Aeschylus</author>
-            <title xml:lang="lat">Aeschyli Tragoediae : cum fabularum deperditarum fragmentis,
-                    poetae vita et operum catalogo</title>
+            <title xml:lang="lat">Aeschyli Tragoediae</title>
                  <editor>Arthur Sidgwick</editor>
             <imprint>
               <pubPlace>Oxford</pubPlace>
               <publisher>Clarendon Press</publisher>
               <date>1902</date>
             </imprint>
           </monogr>
-          <ref target="https://hdl.handle.net/2027/njp.32101064224940">HathiTrust</ref>
+          <ref target="https://hdl.handle.net/2027/njp.32101064224940?urlappend=%3Bseq=64%3Bownerid=27021597765885648-72">HathiTrust</ref>
```

Comment:
I would suggest https://babel.hathitrust.org/cgi/pt?id=njp.32101064224940&view=2up&seq=66

better page and no "owner id" URL append in it

Reply:
I'll change that!